versioned/persist/nontx/src/main/java/org/projectnessie/versioned/persist/nontx/NonTransactionalDatabaseAdapterConfig.java

```diff
@@ -23,6 +23,7 @@ public interface NonTransactionalDatabaseAdapterConfig extends DatabaseAdapterCo
   int DEFAULT_REF_LOG_STRIPES = 8;
   int DEFAULT_REFERENCES_SEGMENT_PREFETCH = 1;
   int DEFAULT_REFERENCE_NAMES_BATCH_SIZE = 25;
+  int DEFAULT_COMMIT_LOG_SCAN_PREFETCH = 25;
```

Comment:
Why not use a large number? It looks like its used only in the context of the unreferenced heads search which is already a memory-heavy operation and will scan all entries, so using large batches may be preference, I guess.

Reply:
I'd rather be on the safe side here - we can experiment with it in "big scenarios" and bump the default based on the experiences. WDYT?

Reply:
sgtm